02-express-tutorial/controllers/people.js

```diff
@@ -0,0 +1,45 @@
+let {people} = require('../data')
+
+const getPeople = (req, res) => {
+    res.status(200).json({ success: true, data: people})
+    console.log('It worked!')
+}
+
+const addPerson = (req, res) => {
+    //console.log(req.body)
+    const {name} = req.body
+    if (name) {
+        people.push({ id: people.length, neame: req.body.name });
```

Comment:
This is not the best approach to define the next id we need to use to create a people. Let's say you have 6 people on your array, when you add a new person, the new ID is 6, but if you remove the person with ID 5 and then add a new person, the new ID will be 6 again, and you will have 2 people with the same id in your array.
To fix this, you can get the last people in your array if the array has at least one person and add the 1 to the id of the last person to define the new ID. If the array is empty, then the new id should be 1.
Here is an example of how to do that:
const nextId =  people.length > 0 ? people[people.length - 1].id + 1 : 1; people.push({ id: nextId, name: req.body.name });